lib/Doctrine/ORM/Cache/DefaultCache.php

```diff
@@ -32,8 +32,8 @@ class DefaultCache implements Cache
     private $uow;
 
      /**
-     * @var \Doctrine\ORM\Cache\CacheFactory
-     */
+      * @var \Doctrine\ORM\Cache\CacheFactory
+      */
     private $cacheFactory;
```

Comment:
I know this (and a lot of others) should also be indented one space before: it will be a future PR